NEWS.md

```diff
@@ -1,4 +1,9 @@
-# lubridate (development version)
+Version 1.9.0.9000
+=============
```

Comment:
This also didn't seem to match the way the NEWS typically looked, so I tweaked it